samples/system-test/quickstart.test.js

```diff
@@ -26,51 +26,55 @@ const dns = new DNS();
 const uuid = require(`uuid`);
 const zoneName = `test-${uuid().substring(0, 13)}`;
 
-test.before(async () => {
-  await dns.createZone(zoneName, {
-    dnsName: `${process.env.GCLOUD_PROJECT}.appspot.com.`,
+describe('QuickStart', () => {
+  before(async () => {
+    await dns.createZone(zoneName, {
+      dnsName: `${process.env.GCLOUD_PROJECT}.appspot.com.`,
+    });
   });
-});
 
-test.after.always(async () => {
-  try {
-    await dns.zone(zoneName).delete();
-  } catch (err) {} // ignore error
-});
-
-test.beforeEach(tools.stubConsole);
-test.afterEach.always(tools.restoreConsole);
+  after(async () => {
+    try {
+      await dns.zone(zoneName).delete();
+    } catch (err) {} // ignore error
+  });
 
-test.cb(`should list zones`, t => {
-  const dnsMock = {
-    getZones: () => {
-      return dns.getZones().then(([zones]) => {
-        t.true(Array.isArray(zones));
+  beforeEach(tools.stubConsole);
+  afterEach(tools.restoreConsole);
 
-        // Listing is eventually consistent, give the indexes time to update
-        setTimeout(() => {
-          try {
-            t.true(console.log.called);
-            t.deepEqual(console.log.getCall(0).args, [`Zones:`]);
-            zones.forEach((zone, i) => {
-              t.deepEqual(console.log.getCall(i + 1).args, [zone.name]);
-            });
-            t.end();
-          } catch (err) {
-            t.end(err);
-          }
-        }, 200);
+  it(`should list zones`, done => {
+    const dnsMock = {
+      getZones: () => {
+        return dns.getZones().then(([zones]) => {
+          assert.ok(Array.isArray(zones));
 
-        return [zones];
-      });
-    },
-  };
+          // Listing is eventually consistent, give the indexes time to update
+          setTimeout(() => {
```

Comment:
Lets make the test an async function.  Then, for setTimeout you can just:

```js
await new Promise(r => setTimeout(r, 200));
```

Reply:
Sure @JustinBeckwith.

Reply:
Change done.